keps/prod-readiness/sig-api-machinery/2340.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 2340
+alpha:
+  approver: "@deads2k"
```

Comment:
Feel free to reassign to me - I will be looking at it anyway.

Reply:
Done